src/pahoMqttPlugin/utils/connectClient.ts

```diff
@@ -74,8 +74,8 @@ export const connectClient = ({
         password: MqttOptions.password ?? '',
         useSSL: MqttOptions.useSSL ?? false,
         uris: [
-          `wss://${MqttOptions.host}:${MqttOptions.port}/mqtt`,
-          `ws://${MqttOptions.host}:${MqttOptions.port}/mqtt`,
+          `wss://${MqttOptions.host}:${MqttOptions.port}${MqttOptions.path ?? '/mqtt'}`,
```

Comment:
Instead of doing `?? '/mqtt'`, you can put the default value of `path` inside the `defaultMqttOptions` object located in `config/constants.ts`.

I know we haven't done it like that in the code above but that is a bad practice in the code. If you want, you can also fix our mistake above.

Reply:
use constant.ts for use default values in MqttOptions, yes is better, I just don't see before.